consul/leader.go

```diff
@@ -153,12 +153,6 @@ func (s *Server) establishLeadership() error {
 		return err
 	}
 
-	// Setup autopilot config if we are the leader and need to
```

Comment:
^ that endpoint still needs an update (it can call get or create prolly and return an error if it can't get a config)